tests/flow/test_constraint.py

```
@@ -990,39 +990,61 @@ def test07_constraint_creation_with_new_relation_attr(self):
+            while True:
+                chunk = sock.recv(4096)
+                if not chunk:
+                    break
+                buf += chunk
+                while b"\n" in buf:
+                    line, buf = buf.split(b"\n", 1)
+                    line = line.rstrip(b"\r").decode("utf-8", errors="replace")
+                    if 'GRAPH.CONSTRAINT' in line and graph_token in line:
+                        self.monitor.append({"command": line})
+        except Exception:
             pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)